packages/core/src/raa/utils.ts

```diff
@@ -124,14 +129,21 @@ export function augmentTopOfferDataWithAnchorPrice({
   }
 }
 
+// Anchor price should be > than 3% from offer price
+const RULE_PERCENTAGE = 0.03
```

Comment:
```suggestion
const ANCHOR_PRICE_DELTA_THRESHOLD_PERCENTAGE = 0.03
```

Just to keep SAPI better than dada :D